app/src/main/java/org/fossasia/openevent/utils/CircleTransform.java

```diff
@@ -24,7 +24,7 @@ public Bitmap transform(Bitmap source) {
             source.recycle();
         }
 
-        Bitmap bitmap = Bitmap.createBitmap(size, size, source.getConfig());
+        Bitmap bitmap = Bitmap.createBitmap(size, size, Bitmap.Config.ARGB_8888);
```

Comment:
@vjs3 Do you have a stack trace for the crash? I am not quite sure if this is right.